templates/worker-policy.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.worker.enabled -}}
-apiVersion: policy/v1
+apiVersion: policy/{{ .Values.worker.apiVersion }}
```

Comment:
there is test failure:

```
Error: INSTALLATION FAILED: 1 error occurred:

	* the server could not find the requested resource

```
I am not sure if you can set the api version like this. Have you tested it locally? Maybe you can try the method like [this one](https://renehernandez.io/snippets/customize-api-version-helm-templates/)

Reply:
@xtremerui thank you for taking a look. The reason I used this method is because for the worker-rolebinding.yaml template the same way got applied: https://github.com/concourse/concourse-chart/blob/b371d83fa31ff1a3c4c3f4ee93103523b2be76c2/templates/worker-rolebinding.yaml#L4

I did test it locally, and it did function. But I'll add anew commit and implement semverCompare.